src/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ find_package(Protobuf REQUIRED)
 find_package(pybind11 REQUIRED)
 find_package(Threads REQUIRED)
 find_package(XRootD REQUIRED)
-find_package(httplib REQUIRED)
+#&&& find_package(httplib REQUIRED)
```

Comment:
I believe this was a problem on main as well. I think and older version was used that had everything find_package was looking for?

Reply:
This works fine in the main branch. The problem was caused by the underlying shift in one of the dependencies of `httplib`. It was addressed in Qserv release: https://github.com/lsst/qserv/releases/tag/2025.4.1-rc2

The problem was analyzed and fixed in this ticket: https://rubinobs.atlassian.net/browse/DM-49862

Reply:
One way this can be addressed is by bringing the change into your development branch by:
```
git cherry-pick 96b68f41bbfcc02cdf78b14f8fa4ad1c7a4ea29f
```